plugins/hypervisors/kvm/src/main/java/com/cloud/hypervisor/kvm/resource/wrapper/LibvirtOvsFetchInterfaceCommandWrapper.java

```diff
@@ -19,6 +19,7 @@
 
 package com.cloud.hypervisor.kvm.resource.wrapper;
 
+import com.google.common.base.Strings;
```

Comment:
Recently we finished a voting defining `commons.lang3` as the default string lib ([VOTE thread](https://lists.apache.org/thread.html/r806cd10b3de645c150e5e0e3d845c5a380a700197143f57f0834d758%40%3Cdev.cloudstack.apache.org%3E) & [RESULT thread](https://lists.apache.org/thread.html/r5898f717d506a0e160b9dc766b65769e2fe2f07f9cba70825843e272%40%3Cdev.cloudstack.apache.org%3E)). The code and the docs will be changed soon, could you already use `commons.lang3` here?

Reply:
@GutoVeronezi done.

Reply:
@weizhouapache thanks!